multigeiger/display.cpp

```diff
@@ -81,6 +81,29 @@ void displayStatusLine(String txt) {
   pu8x8->drawString(0, line, txt.c_str());
 }
 
+int status[STATUS_MAX] = {0, 0, 0, 0, 0, 0, 0, 0};  // current status of misc. subsystems
+
+const char *status_chars[STATUS_MAX] = {
+  " -wW0",  // ST_NODISPLAY, ST_WIFI_OFF, ST_WIFI_CONNECTING, ST_WIFI_CONNECTED, ST_WIFI_ERROR
+  " -bB1",  // ST_NODISPLAY, ST_BT_OFF, ST_BT_CONNECTING, ST_BT_CONNECTED, ST_BT_ERROR
+  " -lL2",  // ST_NODISPLAY, ST_LORA_OFF, ST_LORA_CONNECTING, ST_LORA_CONNECTED, ST_LORA_ERROR
+  " H3",    // ST_NODISPLAY, ST_HV_OK, ST_HV_ERROR
+  " -S4",   // ST_NODISPLAY, ST_SCOMM_IDLE, ST_SCOMM_SENDING, ST_SCOMM_ERROR
+  " -M5",   // ST_NODISPLAY, ST_MADAVI_IDLE, ST_MADAVI_SENDING, ST_MADAVI_ERROR
+  " -T6",   // ST_NODISPLAY, ST_TTN_IDLE, ST_TTN_SENDING, ST_TTN_ERROR
+  " -O7",   // ST_NODISPLAY, ST_TOILET_IDLE, ST_TOILET_SENDING, ST_TOILET_ERROR
```

Comment:
this has to be seen, likely we will not need all these states / not all states make sense.

ideas:
- after boot, everything will display as "blank".
- lowercase == building something up (e.g. connecting)
- uppercase = buildup finished (e.g. connected)
- digit = error in that subsystem

Reply:
Proposition: Have the same amount of letters per status display, with same status (e.g. uppercase) on same position. Fill unused positions with e.g. dot ("."). That way you can merge status changes more easily.

Reply:
Maybe we can not make this completely conistent as these are quite different subsystems.

But I can change NODISPLAY from space to dot.

Reply:
What I meant was to make the position of the states consistent, e.g.
// NODISPLAY, OFF, IDLE, CONNECTING, CONNECTED, ERROR
  " -.wW0",  // ST_NODISPLAY, ST_WIFI_OFF, -, ST_WIFI_CONNECTING, ST_WIFI_CONNECTED, ST_WIFI_ERROR
  " -b.B1",  // ST_NODISPLAY, ST_BT_OFF, ST_BT_IDLE, -, ST_BT_CONNECTED, ST_BT_ERROR
  " -.lL2",  // ST_NODISPLAY, ST_LORA_OFF, -, ST_LORA_CONNECTING, ST_LORA_CONNECTED, ST_LORA_ERROR
  " ...H3",    // ST_NODISPLAY, -, -, -, ST_HV_OK, ST_HV_ERROR
  " -s.S4",   // ST_NODISPLAY, ST_SCOMM_OFF, ST_SCOMM_IDLE, -, ST_SCOMM_SENDING, ST_SCOMM_ERROR
...

That way, further functions, like e.g. inverting an error, could be implemented more consistently

Reply:
One addition, maybe WIFI could indicate the Access Point active with "a" instead of idle...

Reply:
fixed, now always in this order: off 0, ok 1, error 2.

off is same as nodisplay now (and displayed as `.`).

ok is uppercase now, error is numeric.

> 2 is custom, depending on the subsystem.
lowercase is used for connecting.

the network states are a bit different: here lowercase is for idle and enabled, uppercase is for sending.

Reply:
AP status now implemented with "A" in place of "W".